comps/llms/text-generation/tgi/llm.py

```diff
@@ -45,6 +56,18 @@
 async def llm_generate(input: Union[LLMParamsDoc, ChatCompletionRequest, SearchedDoc]):
     if logflag:
         logger.info(input)
+
+    llm_endpoint = DEFAULT_ENDPOINT
+    if input.model and MODEL_CONFIGS and configs_map:
```

Comment:
Can you simplify these code?
Seems like MODEL_CONFIGS is not needed here, and llm_endpoint can get using `configs_map.get(input.model).get("endpoint")`.
The exception should be catched and raised in a 'try except' block.

Reply:
Simplified this code as well